sdk/identity/identity/samples/AzureIdentityExamples.md

```diff
@@ -908,6 +915,215 @@ class RotatingCertificateCredential implements TokenCredential {
+
+> IMPORTANT! The token cache contains sensitive data and MUST be protected to prevent compromising accounts. All application decisions regarding the persistence of the token cache must consider that a breach of its content will fully compromise all the accounts it contains.
+
+#### Using the default token cache
+
+The simplest way to persist the token data for a credential is to to use the default `tokenCachePersistenceOptions`. This will persist and read token data from a shared persisted token cache protected to the current account.
+
+```ts
+import { useIdentityExtension, InteractiveBrowserCredential } from "@azure/identity";
+import { cachePersistenceExtension } from "@azure/identity-cache-persistence";
+
+useIdentityExtension(cachePersistenceExtension);
+
+const credential = new InteractiveBrowserCredential({
+  tokenCachePersistenceOptions: {
+    enabled: true
+  }
+});
+```
+
+#### Using a named token cache
+
+Some applications may prefer to isolate the token cache they use rather than using the shared instance. To accomplish this they can specify the `tokenCachePersistenceOptions` when creating the credential and provide a `name` for the persisted cache instance.
+
+```ts
+import { useIdentityExtension, InteractiveBrowserCredential } from "@azure/identity";
+import { cachePersistenceExtension } from "@azure/identity-cache-persistence";
+
+useIdentityExtension(cachePersistenceExtension);
+
+const credential = new InteractiveBrowserCredential({
+  tokenCachePersistenceOptions: {
+    enabled: true,
+    name: "my_application_name"
+  }
+});
+```
+
+#### Allowing unencrypted storage
+
+By default the token cache will protect any data which is persisted using the user data protection APIs available on the current platform. However, there are cases where no data protection is available, and applications may choose to still persist the token cache in an unencrypted state. This is accomplished with the `allowUnencryptedStorage` option.
+
+```ts
+import { useIdentityExtension, InteractiveBrowserCredential } from "@azure/identity";
+import { cachePersistenceExtension } from "@azure/identity-cache-persistence";
+
+useIdentityExtension(cachePersistenceExtension);
+
+const credential = new InteractiveBrowserCredential({
+  tokenCachePersistenceOptions: {
+    enabled: true,
+    allowUnencryptedStorage: true
+  }
+});
+```
+
+By setting `allowUnencryptedStorage` to true, the credential will encrypt the contents of the token cache before persisting it if data protection is available on the current platform. If platform data protection is unavailable, it will write and read the persisted token data to an unencrypted local file ACL'd to the current account. If `allowUnencryptedStorage` is false (the default), a `CredentialUnavailableError` will be thrown in the case no data protection is available.
```

Comment:
```suggestion
By setting `allowUnencryptedStorage` to true, the credential will encrypt the contents of the token cache before persisting it if data protection is available on the current platform. If platform data protection is unavailable, it will write and read the persisted token data to an unencrypted local file with access permissions restricted to the current user. If `allowUnencryptedStorage` is false (the default), a `CredentialUnavailableError` will be thrown in the case no data protection is available.
```

"ACL'd" is a bit too informal and platform specific.